svc-api/handle/role.go

```diff
@@ -149,8 +149,11 @@ func (r *RoleRPCs) GetRole(ctx iris.Context) {
 		ctx.JSON(&response.Body)
 		return
 	}
-
-	ctx.ResponseWriter().Header().Set("Allow", "GET, PATCH, DELETE")
+	if req.Id == "Administrator" {
```

Comment:
use constant instead of hard code
https://github.com/ODIM-Project/ODIM/blob/development/lib-utilities/common/constants.go#L30